Sources/LLVM/StructType.swift

```diff
@@ -58,14 +58,30 @@ public struct StructType: IRType {
   /// list of values.
   ///
   /// - parameter values: A list of values of members of this structure.
+  ///
+  /// - returns: A value representing a constant value of this structure type.
+  public func constant(values: [IRValue]) -> Constant<Struct> {
```

Comment:
Is it reasonable to assert that the length of the values buffer matches the length of the member types for the struct?